src/components/Layout/Layout.tsx

```diff
@@ -42,7 +52,7 @@ const Layout: FC = () => {
 		};
 	}, []);
 
-	const handleClick = () => {
+	const handleClick = async () => {
 		dispatch(saveEmail(''));
 		dispatch(saveToken(''));
 		dispatch(saveUserId(0));
```

Comment:
I dont like 0 in saveUserId. can we set null?

Reply:
ok, I just testing and forget to return the correct value. I'll fix it.